README.md

```diff
@@ -4,3 +4,12 @@ This repo contains basic information about:
 
 - [Docker images produced by the .NET team](./build-info/docker)
 - [Docker images produced by the Go team](./build-info/microsoft/go-images)
+
+## Issues
+
+Issues are disabled on this repo. If you have any problems with .NET or the contents of this repo, file an issue in one of the following repos:
+
+- **[All .NET GitHub repos](https://github.com/dotnet/core/blob/main/Documentation/core-repos.md)**
+- **[.NET containers](https://github.com/Microsoft/Windows-Containers/issues)**
```

Comment:
```suggestion
- **[.NET containers](https://github.com/dotnet/dotnet-docker/issues/new/choose)**
```

Reply:
Should I add this back for the other two repos then?

Reply:
What other two repos?

Reply:
I fixed the link. Separate question is whether we link directly to the new issue chooser. Go and MAR do not have as detailed issue choosers so I think we should leave it as repo links.

Reply:
Oh, I don't care about linking to the "issues" page or not. I was just intending to correct the repo.